src/devices/Ssd1351/Ssd1351.cs

```diff
@@ -0,0 +1,325 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+using System.Device.Spi;
+using System.Drawing;
+using System.Threading.Tasks;
+using Iot.Device.Ssd1351.Command;
+
+namespace Iot.Device.Ssd1351
+{
+    /// <summary>
+    /// A single-chip CMOS OLED/PLED driver with controller for organic/polymer
+    /// light emitting diode dot-matrix graphic display system. 
+    /// </summary>
+    public partial class Ssd1351 : IDisposable
+    {
+
+        private readonly int _dcPinId;
+        private readonly int _resetPinId;
+        private readonly int _spiBufferSize;
+        private readonly bool _disposeGpioController = false;
+
+        private SpiDevice _spiDevice;
+        private IGpioController _gpioDevice;
+        private ColorDepth _colorDepth;
+        private ColorSequence _colorSequence;
+
+        private const byte SCREEN_WIDTH_PX = 128;
+        private const byte SCREEN_HEIGHT_PX = 128;
+        private const int DefaultSPIBufferSize = 0x1000;
+
+        /// <summary>
+        /// Initializes new instance of Ssd1351 device that will communicate using SPI bus.
+        /// A single-chip CMOS OLED/PLED driver with controller for organic/polymer
+        /// light emitting diode dot-matrix graphic display system. 
+        /// </summary>
+        /// <param name="spiDevice">The SPI device used for communication.</param>
+        /// <param name="gpioController">The GPIO controller used for communication.</param>
```

Comment:
I would actually mention in the description that this controller is the one that provides dataCommandPin and resetPin with their appropiate paramrefs so that it is obvious what this parameter is.

Reply:
@joperezr 
Done in the latest PR update